src/data/finset/locally_finite.lean

```diff
@@ -515,4 +517,23 @@ lemma image_add_right_Ioo (a b c : α) : (Ioo a b).image (+ c) = Ioo (a + c) (b
 by { simp_rw add_comm _ c, exact image_add_left_Ioo a b c }
 
 end ordered_cancel_add_comm_monoid
+
+@[to_additive] lemma prod_Ioi_mul_prod_Ioi_mul_eq_prod_off_diag [fintype ι] [linear_order ι]
```

Comment:
Why does this name mention `Ioi` twice when it appears only once in the statement?